src/jshint.js

```diff
@@ -3282,6 +3282,8 @@ var JSHINT = (function() {
             }
             advance("*");
             g = true;
+          } else {
+            g = false;
```

Comment:
unrelated nit, I really think we should use a different name than `g` for this :)